packages/docker-tests/__tests__/docker.test.ts

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
+//import * as fs from 'fs';
```

Comment:
commented out thing lines provide the (obviously) still failing test for the compile endpoint.